engine/src/main/scala/cromwell/engine/workflow/ShadowWorkflowActor.scala

```diff
@@ -76,7 +77,7 @@ object ShadowWorkflowActor {
     */
   case object WorkflowFailedState extends ShadowWorkflowActorTerminalState
 
-  case class ShadowWorkflowActorData(backendAssignments: Map[Call, String])
+  case class ShadowWorkflowActorData(workflowDescriptor: Option[EngineWorkflowDescriptor])
```

Comment:
I wondered about that. :star: for at least thinking of me the first time :) If you could give it another whirl just to be sure that'd be fantastic.